content/en/docs/ops/integrations/zipkin/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Zipkin
+description: How to integrate with Zipkin.
+weight: 20
```

Comment:
I suspect you'll want different weights on the docs to maybe keep them in alphabetical order. I'm not sure if that's guaranteed in the case of same weights.

Reply:
If all the entries have the same weight, I think they will just be processed in the same order they appear in the folder, which is alphabetical order.